test/potluck_test.rb

```diff
@@ -0,0 +1,78 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/dish'
+require './lib/potluck'
+
+class PotluckTest < Minitest::Test
+  def test_it_exists
+    potluck = Potluck.new("7-13-18")
+
+    assert_instance_of Potluck, potluck
+  end
+
+  def test_it_has_a_date
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal "7-13-18", potluck.date
+  end
+
+  def test_it_has_dishes
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal [], potluck.dishes
+  end
+
+  def test_it_has_entre
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+
+    assert_instance_of Dish, cocktail_meatballs
+  end
+
+  def test_it_adds_dishes
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    potluck.add_dish(couscous_salad)
+
+    assert_instance_of Dish, potluck.dishes[0]
+
+  end
+
+  def test_it_has_multiple_dishes
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+    potluck.add_dish(couscous_salad)
+    potluck.add_dish(cocktail_meatballs)
+
+    assert_instance_of Dish, potluck.dishes[1]
+  end
+
+  def test_it_can_count_all_dishes
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+    potluck.add_dish(couscous_salad)
+    potluck.add_dish(cocktail_meatballs)
+
+    assert_equal 2, potluck.dishes.length
+  end
+
+  def test_it_gets_all_from_category
+    #skip
+
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    summer_pizza = Dish.new("Summer Pizza", :appetizer)
+    roast_pork = Dish.new("Roast Pork", :entre)
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+    candy_salad = Dish.new("Candy Salad", :dessert)
+    potluck.add_dish(couscous_salad)
+    potluck.add_dish(summer_pizza)
+    potluck.add_dish(roast_pork)
+    potluck.add_dish(cocktail_meatballs)
+    potluck.add_dish(candy_salad)
+
+    assert_equal "Couscous Salad", potluck.get_all_from_category(:appetizer).first.name
```

Comment:
Very similar comment to the ones above. You don't need to call .first.name on the return value. Assert exactly what you expect the return value to be. You should expect it to be an array with `Dish` instances.